lib/policies/throttlingRetryPolicy.ts

```diff
@@ -7,13 +7,23 @@ import { HttpOperationResponse } from "../httpOperationResponse";
 import { Constants } from "../util/constants";
 import { delay } from "../util/utils";
 
-type ResponseHandler = (httpRequest: WebResourceLike, response: HttpOperationResponse) => Promise<HttpOperationResponse>;
 const StatusCodes = Constants.HttpConstants.StatusCodes;
+const DEFAULT_RETRY_COUNT = 10;
```

Comment:
10 seems maybe a little high, but I'm not necessarily opposed to it

Reply:
I can use 3, which is also used by the other two retry policies.  I was thinking that `Retry-After` is a bit different than other errors in that the service may have a high chance to service the request if the asked delay is respected.